src/test/java/org/fcrepo/importexport/integration/ExporterIT.java

```diff
@@ -317,6 +321,44 @@ public void testExportCustomPredicatesInbound() throws Exception {
                 createResource(obj1.toString()),
                 createProperty("http://example.org/custom"),
                 createResource(col1.toString())));
+        assertFalse(collectionModel.contains(
+                createResource(other.toString()),
+                createProperty("http://example.org/other"),
+                createResource(col1.toString())));
+    }
+
+    @Test
+    public void testExportInboundFiles() throws Exception {
```

Comment:
This test does not seem to be demonstrating a fix to broken behavior. 
When I checkout the `master` branch and insert nothing from this PR except this test, the build succeeds.
I think the test should be linking a resourceX from outside of the target resourceT hierarchy to test that:
1. The resourceX is exported
2. The model of resourceT does not include mention of resourceX